src/Graph.ts

```diff
@@ -0,0 +1,174 @@
+/**
+ * Transform array to graph
+ */
+
+/**
+ * TYPES
+ */
+
+type node = {
+  x: Number;
+  y: Number;
+  value: Number;
+  isExit: Boolean;
+};
+
+type edge = {
+  from: string;
+  to: string;
+  direction: string;
+};
+
+type Graph = {
```

Comment:
you should export these types - you will definitely need them elsewhere

Reply:
Types have been exported in the [last commit](https://github.com/marmelab/kuba-integration/commit/52c48e332abaf5397cf6debd1512e2111062ba31)